ArdusatSDK.cpp

```diff
@@ -364,8 +369,8 @@ int calculateCheckSum(const char *sensor_name, int num_vals, ...) {
 /*
  * toCSV Output functions
  */
-void _output_buffer_reset() {
-  memset(_output_buffer, 0, 512);
+void _resetOutBuf() {
+  memset(_getOutBuf(), 0, OUTPUT_BUF_SIZE);
```

Comment:
@mrasband - the output buffer is still a pointer here - it should only get allocated 1x during runtime, the first time `_getOutBuf` is called. As @samolds said, the idea here is to be able to switch between using an internal `struct` as our buffer from an optionally included file (from the logging library), or dynamically allocate a `char []` if that library is not included. It's not the cleanest, but it's necessary b/c of limited RAM and the relatively large size of the logging library.